docs/index.rst

```diff
@@ -3,185 +3,287 @@ The confluent_kafka API
 
 A reliable, performant and feature rich Python client for Apache Kafka v0.8 and above.
 
-Clients
+Client API 
+   - :ref:`AdminClient <pythonclient_adminclient>`
+   - :ref:`AvroProducer <avro_producer>`
```

Comment:
People will use these instead of the new Serializing producers/consumers, so I suggest adding " (legacy)" after these, and something like " (Avro, Protobuf, JSON, primitive types, etc)" or whatever catches the eye of the reader.

Also suggest pudding AvroPro and AvroCon at the bottom of this list.